microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md

```diff
@@ -1,25 +1,78 @@
 ---
-title: Windowed vs. visual hosting of WebView2
-description: Deciding whether to have your app use windowed hosting vs. visual hosting of the WebView2 control.  Hosting WebView2 in windowed or visual environments.
+title: Windowed vs. Visual hosting of WebView2
+description: Deciding whether to have your app use Windowed hosting vs. Visual hosting of the WebView2 control.  Hosting WebView2 in windowed or visual environments.
 author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.service: microsoft-edge
 ms.subservice: webview
-ms.date: 10/24/2022
+ms.date: 09/18/2024
 ---
-# Windowed vs. visual hosting of WebView2
+# Windowed vs. Visual hosting of WebView2
 
-There are two options for hosting the Microsoft Edge WebView2 control in your app: windowed hosting and visual hosting.
+<!-- outline Sep 28, 2024:
+=======================================
+## Scenarios for selecting the hosting approach
+#### Windowed hosting: For displaying content quickly and easily
+###### Advantages
+###### Disadvantages
+
+#### Window to Visual hosting: Similar experience as Windowed hosting with added benefits and tradeoff
+###### Advantages
+###### Disadvantages
+
+#### Visual hosting: For more granular control over layout
+###### Advantages and disadvantages
 
-If you use windowed hosting, which is a good starting point for most apps, you don't need to read this article.  If you want to provide a more custom user experience (UX), and want to use visual hosting, read this article.
+=======================================
+## Compatibility and constraints
+#### Operating systems
+#### Rendering WebView2 in framework and non-framework apps    [APIs tabset]
+
+=======================================
+## Windowed hosting and Window to Visual hosting
+#### Benefits of Windowed hosting
+#### Benefits of Window to Visual hosting
+#### APIs for Windowed hosting and Window to Visual hosting
+#### Window management                            [APIs tabset]
```

Comment:
```suggestion
###### Window management                            [APIs tabset]
```
can't demote its children / following two sections but can make this change work

Reply:
demoted API headings to h6, incl in outline